.gitignore

```diff
@@ -113,3 +113,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#for vscode
+.vscode
```

Comment:
Minor: If you add `Closes #8` in the PR description, merging this PR will automatically close that issue.

Reply:
Should I write Closes <#issue_no> only in the description or writing along some other text will also work?

Reply:
I think GitHub looks for `Closes #no` in the description. You can write other text in it too.